util/logging.c

```diff
@@ -17,14 +17,15 @@
 
 #include "logging.h"
 #include "sighdl.h"
+#include "nvme-print.h"
 
 struct submit_data {
 	struct timeval start;
 	struct timeval end;
 };
 
 int log_level;
-static bool dry_run;
+static bool dry_run_state;
```

Comment:
... and then you can just directly use the `dry_run`. so you can delete this definition.